content/en/docs/application-store/built-in-apps/harbor-app.md

```diff
@@ -6,5 +6,101 @@ description: 'How to use built-in Harbor registry'
 
 weight: 2242
 ---
+From the [Introduction](../../_index) section, you know there was uncounted application could be installed by helm. [kubesphere\'s App Store](https://charts.kubesphere.io/main/) also added some popular application.
 
-TBD
+This tutorial walks you through an example of how to deploy [Harbor](https://goharbor.io/) with several click in kubesphere.
+
+## Prerequisites
+
+- Please make sure you [enable the OpenPitrix system](https://kubesphere.io/docs/pluggable-components/app-store/). We will deploy Harbor from the App Store.
+- You need to create a [workspace, a project, and a user account](https://kubesphere.io/docs/quick-start/create-workspace-and-project/) for this tutorial. The account needs to be a platform regular user and to be invited as the project operator with the `operator` role. In this tutorial, you log in as `project-operator` and work in the project `demo` in the workspace `demo-wp`.
+
+## Hands-on Lab
+
+### Common steps
+
+1. To begin with, you deploy Harbor from the App Store. Please make sure you are landing on the **Overview** page of the project `demo`.
+
+![overview_of_demo_project](/images/docs/appstore/harbor/overview_of_demo_project.png)
+
+2. List your deployed apps. If it is empty, click the button `Deploy New Application`.
+
+![new_app_deploy](/images/docs/appstore/harbor/new_app_deploy.png)
+
+3. Choose harbor template `From App Store`.
+
+![choose_app_from_store](/images/docs/appstore/harbor/choose_app_from_store.png)
+
```

Comment:
The steps from 1 to 3 are unnecessary, why not deploy an App from the App Store entry at the left top corner?

![Uploading image.png…]()

Reply:
ok